base/tests/test_forms.py

```diff
@@ -415,6 +416,7 @@ async def test_submit_to_healthconnect(self):
         """
         base.actions.actions.config.HEALTHCONNECT_URL = "https://healthconnect"
         base.actions.actions.config.HEALTHCONNECT_TOKEN = "token"
+        base.actions.actions.config.LANGUAGE = "eng"
```

Comment:
Since this defaults to `eng`, I don't think this is needed. What we could do is change this to another language, and assert that the language is set correctly in the data

Reply:
Hmm I had that because I reused the environment variable `LANG`, I changed that now. thanks